### PR TITLE
Fits output vf

### PR DESCRIPTION
**Pull request by hbrincon**
Writes both the VoidFinder and V2 output to single FITS files. Additional information about the voidfinding input parameters and performance are saved to the FITS headers (addresses issue #76). VoidFinder and V2 have been modified to accept custom column names for the galaxy input file (e.g. so that a file with a magnitude column named ELPETRO_ABSMAG_R or magnitude_r will be mapped to rabsmag). The test suite has been modified to accommodate the new output files. Parts of the V2 classes.py file have been commented for readability. V2 can now read in txt and h5 files in addition to fits files

### CODE REVIEW

**Comment by hbrincon** [comment]
Combines output files of VoidFinder into a single fits file with catalog info in the header. Addresses issue #76

**Comment by hbrincon** [comment]
Now writes both the VoidFinder and V2 output to single FITS files. Addresses issue #76

**Review by kadglass — CHANGES_REQUESTED** [review]
A thorough rewrite of the file I/O so that all output is saved in a single fits file for a given algorithm.  I have a few minor comments/requests for some of the changes before approving the pull request.

File: python/vast/voidfinder/preprocessing.py
[the remark this answers is not in the document]
Comment (hbrincon) [reply python/vast/voidfinder/preprocessing.py:1]: All the file name options (no mag cut, keep isolated, distance metric) are now reported in the primary FITS header, and so I thought we could remove them from the file name. Do we want to keep the option to name the file after these choices? I can add then back if so.

File: python/vast/voidfinder/voidfinder.py
[the remark this answers is not in the document]
Comment (hbrincon) [reply python/vast/voidfinder/voidfinder.py:1]: The maximal spheres are still being saved. The `maximal_spheres_table` table and the `myvoids_table` table are passed to the file output function `save_output_from_find_voids` for the maximals and holes respectively.

File: example_scripts/SDSS_VoidFinder_dr7.py
[the remark this answers is not in the document]
Comment (hbrincon) [reply example_scripts/SDSS_VoidFinder_dr7.py:1]: See the below comment. I can add `rm_isolated=rm_isolated` back if desired

File: example_scripts/classifyEnvironment.py
[the remark this answers is not in the document]
Comment (hbrincon) [reply example_scripts/classifyEnvironment.py:1]: I believe that flag is what it's currently called in the master branch https://github.com/DESI-UR/VAST/blob/a3a5fac39975734cf5ebd9658921f22632f2c1d7/python/vast/voidfinder/hole_combine.py#L283

I changed the "voidID" label in the comment in `classifyEnvironment.py` to match the current label. Do we want to change the label back to "'voidID'" from the current master branch name?

[the remark this answers is not in the document]
Comment (kadglass) [reply example_scripts/classifyEnvironment.py:1]: Double-checking in the public SDSS DR7 catalog, the column in question is named `void`.  Let's use this label for the void number associated with each hole.

**Review by kadglass — APPROVED** [review]
All void "flag" columns changed to "void".

**Review by kadglass — APPROVED** [review]
All version numbers updated, and updates are documented in the changelog.

**Pull request merged**